modules/microshift-otel-install.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+//  microshift_running_apps/microshift-observability-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-install_{context}"]
+= Installing and enabling {microshift-short} Observability
+
+You can install {microshift-short} Observability at any time, including during the initial {microshift-short} installation.
+
+.Procedure
+. Install the `microshift-observability` RPM by entering the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-observability -y
```

Comment:
```suggestion
$ sudo dnf install microshift-observability
```
I'd suggest removing `-y` for consistency because it's missing also when installing other RPMs